### PR TITLE
plasma-infra: Improve lerna bootstrap time execution [Part 2]

### PR DESCRIPTION
**Pull request by Yakutoc**
Продолжение #574 

## Lint 

Пропускаем workflow если нет изменений в пакетах - `@salutejs/plasma-{ui,web,b2c,temple}`

Запускаем `px lerna bootstrap` с флагом `--since`

## Unit test

Запускаем с флагом `--since`. Здесь логика работы такая же как и с "Typescript Coverage"  в #574 

## Theme Builder

Убран лишний шаг связанный с подготовкой репозитория, минус 1-1.30 мин от времени выполнения.

## Cypress

Запускаем **bootstrap** с определенным набором `scope = {b2c, web, core, hope, icons, typo, tokens, cy-utils, ui}` 

Время исполнения  **bootstrap** варьируется в среднем в пределах 5 мин. Конечно бывают выбросы и до 7 мин.         

Screenshot 2023-06-26 at 12 50 46

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.575.5376012008.swift)  
[color_asdk_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.575.5376012008.kt)  
[color_asdk_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.575.5376012008.ts)  
[color_sberHealth_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.575.5376012008.swift)  
[color_sberHealth_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.575.5376012008.kt)  
[color_sberHealth_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.575.5376012008.ts)  
[color_sbermarket_business_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.575.5376012008.swift)  
[color_sbermarket_business_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.575.5376012008.kt)  
[color_sbermarket_business_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.575.5376012008.ts)  
[color_sbermarket_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.575.5376012008.swift)  
[color_sbermarket_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.575.5376012008.kt)  
[color_sbermarket_metro_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.575.5376012008.swift)  
[color_sbermarket_metro_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.575.5376012008.kt)  
[color_sbermarket_metro_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.575.5376012008.ts)  
[color_sbermarket_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.575.5376012008.ts)  
[color_sbermarket_selgros_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.575.5376012008.swift)  
[color_sbermarket_selgros_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.575.5376012008.kt)  
[color_sbermarket_selgros_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.575.5376012008.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.575.5376012008.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.575.5376012008.kt)  
[color_sbermarket_wlbusiness_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.575.5376012008.ts)  
[color_sberonline_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.575.5376012008.swift)  
[color_sberonline_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.575.5376012008.kt)  
[color_sberonline_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.575.5376012008.ts)  
[color_sberprime_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.575.5376012008.swift)  
[color_sberprime_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.575.5376012008.kt)  
[color_sberprime_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.575.5376012008.ts)  
[shadow_sbermarket_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.575.5376012008.ts)  
[typo_mage_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.575.5376012008.swift)  
[typo_mage_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.575.5376012008.kt)  
[typo_mage_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.575.5376012008.ts)  
[typo_plasma_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.575.5376012008.swift)  
[typo_plasma_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.575.5376012008.kt)  
[typo_plasma_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.575.5376012008.ts)  
[typo_ruler_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.575.5376012008.swift)  
[typo_ruler_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.575.5376012008.kt)  
[typo_ruler_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.575.5376012008.ts)  
[typo_sage_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.575.5376012008.swift)  
[typo_sage_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.575.5376012008.kt)  
[typo_sage_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.575.5376012008.ts)  
[typo_sbermarket_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.575.5376012008.swift)  
[typo_sbermarket_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.575.5376012008.kt)  
[typo_sbermarket_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.575.5376012008.ts)  
[typo_soulmate_ios-swift--canary.575.5376012008.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.575.5376012008.swift)  
[typo_soulmate_kotlin--canary.575.5376012008.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.575.5376012008.kt)  
[typo_soulmate_react-native--canary.575.5376012008.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.575.5376012008.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.217.1-canary.575.5376012008.0
  npm install @salutejs/cra-template-plasma-basic-template@2.3.1-canary.575.5376012008.0
  npm install @salutejs/plasma-colors@0.7.1-canary.575.5376012008.0
  npm install @salutejs/plasma-core@1.122.1-canary.575.5376012008.0
  npm install @salutejs/plasma-hope@1.217.1-canary.575.5376012008.0
  npm install @salutejs/plasma-icons@1.151.1-canary.575.5376012008.0
  npm install @salutejs/cra-template-plasma-shop-template@2.2.3-canary.575.5376012008.0
  npm install @salutejs/plasma-temple@1.166.1-canary.575.5376012008.0
  npm install @salutejs/plasma-tokens-b2b@1.22.1-canary.575.5376012008.0
  npm install @salutejs/plasma-tokens-b2c@0.31.1-canary.575.5376012008.0
  npm install @salutejs/plasma-tokens-core@0.2.1-canary.575.5376012008.0
  npm install @salutejs/plasma-tokens-web@1.37.1-canary.575.5376012008.0
  npm install @salutejs/plasma-tokens@1.56.1-canary.575.5376012008.0
  npm install @salutejs/plasma-typo@0.34.1-canary.575.5376012008.0
  npm install @salutejs/plasma-ui@1.198.1-canary.575.5376012008.0
  npm install @salutejs/plasma-web@1.217.1-canary.575.5376012008.0
  npm install @salutejs/plasma-cy-utils@0.63.1-canary.575.5376012008.0
  npm install @salutejs/plasma-sb-utils@0.120.1-canary.575.5376012008.0
  npm install @salutejs/plasma-tokens-utils@0.27.1-canary.575.5376012008.0
  # or 
  yarn add @salutejs/plasma-b2c@1.217.1-canary.575.5376012008.0
  yarn add @salutejs/cra-template-plasma-basic-template@2.3.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-colors@0.7.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-core@1.122.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-hope@1.217.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-icons@1.151.1-canary.575.5376012008.0
  yarn add @salutejs/cra-template-plasma-shop-template@2.2.3-canary.575.5376012008.0
  yarn add @salutejs/plasma-temple@1.166.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-tokens-b2b@1.22.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-tokens-b2c@0.31.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-tokens-core@0.2.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-tokens-web@1.37.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-tokens@1.56.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-typo@0.34.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-ui@1.198.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-web@1.217.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-cy-utils@0.63.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-sb-utils@0.120.1-canary.575.5376012008.0
  yarn add @salutejs/plasma-tokens-utils@0.27.1-canary.575.5376012008.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-575/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-575/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-575/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-575/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-575/

File: .github/workflows/cypress-common.yml [hunk 1]
```diff
@@ -34,7 +42,7 @@ jobs:
           npm i --no-progress react@17 react-dom@17 @types/react@17.0.40 @types/react-dom@17 --prefix="./packages/plasma-web"
 
       - name: Lerna bootstrap
-        run: npx lerna bootstrap
+        run: npx lerna bootstrap --scope=@salutejs/plasma-${{env.SCOPE}} --scope="@salutejs/platform-test"
```
Comment (neretin-trike) [inline .github/workflows/cypress-common.yml:45]: а тут это ещё надо?

**Review by neretin-trike — APPROVED** [review]

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**